inbm/dispatcher-agent/dispatcher/source/source_command.py

```diff
@@ -120,7 +120,7 @@ def _handle_app_source_command(
             logger.info(f"Optional GPG key parameters not present in manifest")
         filename = parsed_head.get_children("applicationSource/remove/repo")["filename"]
         application_source_manager.remove(
-            ApplicationRemoveSourceParameters(file_name=filename, gpg_key_name=keyname)
+            ApplicationRemoveSourceParameters(source_list_file_name=filename, gpg_key_name=keyname)
```

Comment:
**Suggestion:** Refactor the repeated code for parsing GPG key parameters into a separate function to improve code maintainability. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            keyname, filename = parse_gpg_key_parameters(parsed_head, "remove")
            application_source_manager.remove(
                ApplicationRemoveSourceParameters(source_list_file_name=filename, gpg_key_name=keyname)
            )
```